taskcluster/scriptworker-task-github.yml

```diff
@@ -25,9 +25,11 @@ build:
   artifacts_deps:
     python:
       - "darwin-amd64-cpu-opt"
+      - "darwin-amd64-ctc-opt"
```

Comment:
This is good but not enough: i've indicated that you also need to add those in the list of `dependencies` above.

Reply:
I will add it